vms/avm/txs/parser.go

```diff
@@ -130,14 +127,6 @@ func (p *parser) ParseGenesisTx(bytes []byte) (*Tx, error) {
 	return parse(p.gcm, bytes)
 }
 
-func (p *parser) InitializeTx(tx *Tx) error {
-	return initializeTx(p.cm, tx)
-}
-
-func (p *parser) InitializeGenesisTx(tx *Tx) error {
-	return initializeTx(p.gcm, tx)
-}
```

Comment:
`Tx` has an `Initialize` function and we already expose the `codec` - so these functions aren't needed. Additionally, `InitializeTx` was only used in tests.